src/main/java/net/glowstone/generator/objects/trees/AcaciaTree.java

```diff
@@ -64,7 +64,7 @@ public boolean generate() {
                 twistCount--;
             }
 
-            Material material = blockTypeAt(centerX, loc.getBlockY() + y, centerZ);
```

Comment:
These changes are also not correct, right? (Also in the following files, all the blockAt method renames)